tests/pages/base.py

```diff
@@ -2,14 +2,27 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from pypom import Page, Region
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.select import Select
 
-from pypom import Page, Region
 from pages.regions.newsletter import NewsletterEmbedForm, LegacyNewsletterEmbedForm
 
 
-class BasePage(Page):
+class ScrollElementIntoView:
+
+    def scroll_element_into_view(self, strategy, locator):
```

Comment:
This adds a global `scroll_element_into_view()` function, which offsets the height of the sticky navigation. This should hopefully allows tests to avoid getting caught out by the sticky navigation and trying to click on buttons positioned underneath the top element 🤞

Reply:
🔥